scheduler/src/cook/mesos/unscheduled.clj

```diff
@@ -161,6 +162,16 @@
       ["You are currently rate limited on how many jobs you launch per minute."
        {:max-jobs-per-minute tokens-replenished-per-minute}])))
 
+(defn- check-hook-filter
+  "Return the appropriate error message if a user's job is unscheduled because the hook plugin
+  has blocked the launch."
+  [job]
+  (let [{:keys [status message]} (launch-hooks/get-filter-status job)]
+    (when (= status :deferred)
+      ["The hook plugin is blocking the job launch."
```

Comment:
Do we need this message? We're getting a better message from the plugin, right?

Reply:
IMO, yes. Its good as it gives context. I want the message to include the triplite of 'hooks' <implementation type implementing the hook> and <message from the implementation>.

Reply:
triplite of hoooks, [class-of-plugin], and [message from that plugin].

Test; does github strip things in angle braces? <some test text>

Reply:
triplet?

Reply:
triplet